src/ngx_http_upsync_module.c

```diff
@@ -3221,7 +3234,8 @@ ngx_http_parser_execute(ngx_http_upsync_ctx_t *ctx)
         return;
     }
 
-    if (ngx_strncmp(state.status, "OK", 2) == 0) {
+    if (ngx_strncmp(state.status, "OK", 2) == 0 
+            || ngx_strncmp(state.status, "Bad", 3) == 0) {
```

Comment:
Even if this pass Bad body to parse_json, it is ok because json is key-value format.

But if we move state.status to ngx_http_upsync_parse_init, is it better? Because in parse_init function it is independent by conf_server, and it can distinguish consul and etcd, then decision pass what status body to the parse_json.

What do you think? @CallMeFoxie 

Anyway, thanks very much.